components/brave_wallet_ui/common/slices/api.slice.ts

```diff
@@ -1203,7 +1206,7 @@ export function createWalletApi (
         GetAccountTokenCurrentBalanceArg
       >({
         queryFn: async (
-          { account, token },
+          { coin, address, token },
           { dispatch },
```

Comment:
This is used for existing wallet's accounts and for possible hw accounts, so no AccountId interface. Maybe there should be wrappers two different backend calls - one for AccountId another for coin+address. To be fixed somehow for bitcoin accounts which don't have address.